main/FatesHistoryInterfaceMod.F90

```diff
@@ -287,7 +287,7 @@ module FatesHistoryInterfaceMod
   integer :: ih_fire_fuel_sav_si
   integer :: ih_fire_fuel_mef_si
   integer :: ih_sum_fuel_si
-  integer :: ih_fragmentation_scalar_si
+  integer :: ih_fragmentation_scaler_si
```

Comment:
i stared at this for about a minute before I could tell what changed

Reply:
yeah, the commit message says that I fixed a typo, but really i unfixed a typo to allow for consistency.  or maybe not -- these things are all scalars that scale rates, so I've never been completely sure if they should be scalars or scalers. anyway, scaler is widespread in the code and am leaving it as such here, despite my suspicion that scalar is slightly preferable.